### PR TITLE
Add helper script for image deployment over LAVA [REVPI-2582]

### PR DESCRIPTION
**Pull request by ferdrich-kunbus**

### CODE REVIEW

File: worker/lava_recovery_cmd.sh [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    sleep 0.5
+
+    rc_set_relais "$IPADDR" "$PORT" "$POWER_RELAIS" 1
+    sleep 0.5
+}
+
+case "$CMD" in
+    on) recovery_start ;;
+    off) recovery_exit ;;
+    *) echo "only on and off are supPORTed as parameter for recovery mode" && exit ;;
```
Comment (RamiGspo) [inline worker/lava_recovery_cmd.sh]: "supPORTed"?
Is this on purpose?

File: worker/lava_recovery_cmd.sh [hunk 2]
```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env bash
+
+if [[ $# -ne 6 ]]; then
+  echo "help: $0 <Relaiscard-IP> <Relaiscard-Port> <Power-Relais-No> <USB-Relais-No> <USB-Location> <on|onff>"
```
Comment (RamiGspo) [inline worker/lava_recovery_cmd.sh]: Maybe **<on|off>** ?

File: worker/lava_recovery_cmd.sh (hunk 1 above)
Comment (ferdrich-kunbus) [reply worker/lava_recovery_cmd.sh]: He he, no. :)
I replaced the lower case variable "port" with its uppercase variant and vim also catched "port" in supported and I didn't notice it.

File: worker/lava_recovery_cmd.sh (hunk 2 above)
Comment (ferdrich-kunbus) [reply worker/lava_recovery_cmd.sh]: Absolutely

File: worker/lava_recovery_cmd.sh [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+
+  if [ ! -b "/dev/$usb_disk" ]; then
+    echoerr "no storage device found for USB device $USB_LOC"
+    exit
+  fi
+
+  echoinfo "RPi mass storage added as /dev/$usb_disk"
+  echoinfo "Looking for a running container"
+
+  # now get this shit into the container, get containers,  one per line and only running ones
```
Comment (RamiGspo) [inline worker/lava_recovery_cmd.sh]: Do we need to write bad words?

File: worker/tools/relaiscard.sh [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+#   $3 which relais to set as number (1 to max relais number)
+#   $4 value to set relais to as number (0, 1)
+rc_set_relais () {
+  if [[ $# != 4 ]]; then
+    echo "Error: wrong number of function parameters"
+    return 1
+  fi
+
+  __send_cmd "$1" "$2" "SR $3 $(__number_to_onoff "$4")" &>/dev/null
+
```
Comment (RamiGspo) [inline worker/tools/relaiscard.sh:24]: Sometimes there is a space before the return and sometimes there is not... 
If an error occurs, there is no space. But if it happens, there is not...

Do Coding Rules exist for us?

Comment (ferdrich-kunbus) [reply worker/tools/relaiscard.sh:24]: There's no coding rule for that, although it is heavily discussed on some public platforms in the internet. Normally I add a newline before a return at the end of a function for readability, but not somewhere in the function itself. The exception is __send_cmd, where the content of the function is so small, that it seems to be waste to add a newline before the return.

Some people count empty lines in functions as code smell and as reason to refactor the function to smaller functions. There is a point, where a function is so small, that further splitting of the content makes almost no sense, but adding an empty line may increase the readability.

File: worker/lava_recovery_cmd.sh (hunk 3 above)
Comment (ferdrich-kunbus) [reply worker/lava_recovery_cmd.sh]: Sometimes, but I will remove it here.

**Review by RamiGspo — APPROVED** [review]
LGTM

**Pull request merged**